constants/api.js

```diff
@@ -0,0 +1 @@
+export const baseUrl = 'http://localhost:5710/';
```

Comment:
we'll have to change this later to check if you're on development or not and switch to the prod url if so